src/main/kotlin/org/taktik/icure/services/external/rest/v1/controllers/extra/TimeTableController.kt

```diff
@@ -84,6 +84,7 @@ class TimeTableController(
 
 				//Create a timeTableItem
 				val timeTableItem = TimeTableItem(
+					rrule =null,
```

Comment:
If rrule replaces days & recurrenceTypes field, to be fully compatible to what you had before, don't you need to include "monday" in the rrule string in a standard way ?

Reply:
We can't have a default value for rrule as we need to check if rrule is null to implement backward compatibility: switch to the legacy logic based on days and recurrenceType when rrule is missing.

Reply:
Ok